templates/tutorials/tutorial.html

```diff
@@ -208,4 +208,31 @@ <h2 class="p-heading--three">{{ loop.index }}. {{ section["title"]}}</h2>
     window.location.hash == "#welcome";
   }
 </script>
+
+<script>
+  (function() {
+    var polls = document.querySelectorAll('.poll');
+
+    polls.forEach(function(poll) {
```

Comment:
Was gonna mention that `forEach` doesn't work on nodeLists in IE11, so you could replace those calls in this script with a workaround e.g:

```suggestion
    [].forEach.call(polls, function (poll) {
```

But tutorials don't load at all in IE11, there's a JS error involving `fetch`, not sure whether to raise that as a separate issue.

Reply:
Thanks @sowasred2012 - I'm too used to babel taking care of that stuff!

I didn't see the fetch error but tutorials doesn't use fetch? Anyway I've fixed up the nodelist foreach and all works now.

Reply:
Oh weird! Yeah I don't see it anymore either - maybe I imagined it? Looks good 👍